python/mlc_chat/serve/engine.py

```diff
@@ -108,6 +109,81 @@ def __init__(
         # Load the engine with the input models and KV cache config.
         self._reload(models, kv_cache_config)
 
+    def generate(
```

Comment:
@MasterJH5574 are there any plans to make generate async? the data movement between devices is a lot of IO considering a lot of incoming requests